### PR TITLE
Updates instructions on how to import models

### PR DESCRIPTION
**Pull request by joshdevins**
This update will point directly to the eland API which has an example and full list of supported models. This is the right place to document limitations as this is where it's implemented.

### CODE REVIEW

**Comment by joshdevins** [comment]
@sethmlarson We want to link to some API docs in eland. Is there a way to get a static link to the "latest"? I don't really want to embed the version number in there like:

https://eland.readthedocs.io/en/7.10.1b1/reference/api/eland.ml.MLModel.import_model.html#eland-ml-mlmodel-import-model

File: docs/en/stack/ml/df-analytics/ml-trained-models.asciidoc [hunk 1]
```diff
@@ -143,15 +143,9 @@ Elasticsearch cluster, for example by adding nodes.
...
-The example uses Python to train, save and import the model. However, you can
-use any client as long as the model conforms to the 
-https://github.com/elastic/ml-json-schemas[schema].
-
-// This blog post is a step by step description of how to create a random forest
-// classifier {ml} model outside of {es} by using Python, load it into {es},
-// then operationalize it with ingest pipelines.
+trained by Elastic {dfanalytics}. https://eland.readthedocs.io/[`eland`] supports several types of models from
+libraries like scikit-learn[https://scikit-learn.org/stable/] and XGBoost[https://xgboost.readthedocs.io/].
+In `eland` the
```
Comment (sethmlarson) [inline docs/en/stack/ml/df-analytics/ml-trained-models.asciidoc:148]: Module name is `eland` however the project title is capitlized "Eland"

**Comment by joshdevins** [comment]
Ok this change is now really small! Thanks @sethmlarson  , I've pointed to the client docs and mentioned to refer to the latest eland docs for more details.

**Comment by joshdevins** [comment]
(@szabosteve please squash and merge all those commits when you end up merging the PR!)

**Review by szabosteve — APPROVED** [review]
LGTM! Thanks!

**Comment by szabosteve** [comment]
**Backport commits**:
7.x: 0d5efc0
7.11: 5321b4d

**Pull request merged**